test/interception.basic1.test.js

```diff
@@ -227,8 +238,23 @@ addTestCases((name, transferMethod, setupFunc, setupArgs = []) => {
                 });
             });
 
-            const cookies = await page.cookies();
-            assert.strictEqual(cookies.length, 0);
+            let cookies = await page.cookies();
+
+            // Assert the sameSite=None and Secure: true attributes for sFrontToken as it will be set for both the cookie and header based auth
+            const sFrontToken = cookies.find(cookie => cookie.name === "sFrontToken");
+            assert(sFrontToken.sameSite === "None");
+            assert(sFrontToken.secure === true);
+
+            // The following cookies may or may not be set depending on header or cookie based auth.
+            // We will check the presence of the cookies before asserting the attributes
+            const frontendCookies = ["sAntiCsrf", "st-last-access-token-update", "st-access-token", "st-refresh-token"];
+            frontendCookies.forEach(cookieName => {
+                const cookie = cookies.find(cookie => cookie.name === cookieName);
+                if (cookie) {
+                    assert(cookie.sameSite === "None");
+                    assert(cookie.secure === true);
+                }
+            });
```

Comment:
Please make these into an explicit check, we know which we expect to be there and which we don't based on the  transfermethod

Reply:
Didn't realize we had access to `transferMethod` in the test. I have updated the test to make explicit checks